libraries/LITTLEFS/src/LITTLEFS.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright 2015-2020 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+static constexpr const char LFS_NAME[] = "spiffs";
+
+#include "vfs_api.h"
+
+extern "C" {
+#include <sys/unistd.h>
+#include <sys/stat.h>
+#include <dirent.h>
+#include "esp_littlefs.h"
+}
```

Comment:
you can add the following lines here to clear the issue with termios:
```cpp
#undefine B110
#undefine B1000000
```

Reply:
Can you fix it from your side, please :) I am almost loosing my job trying to understand how this GIT works :)